src/libs/Navigation/Navigation.js

```diff
@@ -199,6 +199,20 @@ function getActiveRoute() {
     return '';
 }
 
+/** Returns the active route name
+ * @returns {String}
+ * */
+
+function getActiveRouteName() {
+    const currentRoute = navigationRef.current && navigationRef.current.getCurrentRoute();
+    const currentRouteHasName = lodashGet(currentRoute, 'name', false);
+    if (!currentRouteHasName) {
```

Comment:
FWIW, I think it's perfectly fine for this function to just do return String or undefined, but only String is also okay. (Generally I like having undefined for the "not set" case so it's simple to write conditionals for). Was there anything in particular that pushed you in this direction?